elyra/pipeline/component_parser_airflow.py

```diff
@@ -144,40 +148,26 @@ def _parse_properties(self, registry_entry, component_definition, component_clas
             if match:
                 type = match.group(1).strip()
 
-            properties.append(ComponentProperty(ref=arg,
-                                                name=arg,
-                                                type=type,
-                                                value=default_value,
-                                                description=description,
-                                                control_id=control_id))
+            properties.append(ComponentParameter(ref=arg,
```

Comment:
why are we calling this ref? and not id? I know this might be overriding things, etc but we should be consistent...

Reply:
I have a local change to rename this back to id, but will not push before discussing it.